.goreleaser.yaml

```diff
@@ -56,13 +45,13 @@ partial:
   by: target
 
 archives:
```

Comment:
question: A bit rusty on this, what was `archives`?

Reply:
Its for customizing the files that are included in the final build archive.

Reply:
So according to this we will only have a `slim` version on github release page and not the main (all features one?). While in the defra container file we have the one with all features? Or maybe I am misunderstanding this

Reply:
There are both slim and normal archives that will be on GitHub releases. The Docker will be the same as the normal build.

Reply:
Okay perfect thanks